tensorflow_serving/servables/tensorflow/util.cc

```diff
@@ -47,6 +47,48 @@ auto* example_count_total = monitoring::Counter<1>::New(
     "/tensorflow/serving/request_example_count_total",
     "The total number of tensorflow.Examples.", "model");
 
+// Metrics by model
+auto* model_request_count_total = monitoring::Counter<1>::New(
+    "/tensorflow/serving/model_request_count",
+    "The total number of requests.", "model");
+
+auto* model_request_fail_count_total = monitoring::Counter<1>::New(
+    "/tensorflow/serving/model_request_fail_count",
+    "The total number of faled requests.", "model");
+
+auto* model_latency_total = monitoring::Counter<1>::New(
```

Comment:
why do we need this? can't aggregator/collectors sum this up from the latency distribution (next counter)?